tutorials/s3output/README.md

```diff
@@ -0,0 +1,7 @@
+Sample application to show how to use the S3OutputModule to upload files into Amazon S3 Bucket.
+
+Operators in sample application are as follows:
+1) FSInputModule which reads files from file systems HDFS/NFS and emits FileMetadata, BlockMetadata, BlockBytes.
+2) S3OutputModule which uploads the files into S3 Bucket using multi-part upload feature.
+
+Please configure the module properties which is under src/main/resources/META-INF/ before launching the application. 
```

Comment:
Some properties are left blank in properties.xml; it will be helpful to discuss those here including some sample values (e.g. for outputDirectory) or add a pointer to where those properties are documented or both.